cinema/serializers.py

```diff
@@ -43,6 +54,7 @@ class MovieListSerializer(MovieSerializer):
     actors = serializers.SlugRelatedField(
         many=True, read_only=True, slug_field="full_name"
     )
+    image = MovieImageSerializer
```

Comment:
The `image` field in `MovieListSerializer` should instantiate `MovieImageSerializer`. Change `image = MovieImageSerializer` to `image = MovieImageSerializer()`.